### ISSUE TITLE
SC2183: Grammar: "1 variables" instead of "1 variable"

### ISSUE THREAD

**Issue by Jonta** [koalaman/shellcheck#2400]
- SC2183
- 0.7.1
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh
printf "%s\n"

```

#### Here's what shellcheck currently says:

       ^-- SC2183 (warning): This format string has 1 variables, but is passed 0 arguments.

#### Here's what I wanted or expected to see:

       ^-- SC2183 (warning): This format string has 1 variable, but is passed 0 arguments.

**Comment by koalaman** [comment]
I additionally applied the same logic to "arguments" in ea4e0091c. Thanks!

**Comment by koalaman** [comment]
Fixed by @ygeyzel. Thanks!

**Comment by plambert** [comment]
Would it make sense to also pluralize the word "arguments" the same way as "variables?"
